easybuild/easyconfigs/p/pftoolsV3/pftoolsV3-3.2.12-GCCcore-11.2.0.eb

```
@@ -0,0 +1,63 @@
+easyblock = 'CMakeMake'
+
+name = 'pftoolsV3'
+version = '3.2.12'
+
+homepage = 'https://github.com/sib-swiss/pftools3'
+description = """A suite of tools to build and search generalized profiles (protein and DNA)."""
+software_license = 'LicenseGPLv2'
+
+toolchain = {'name': 'GCCcore', 'version': '11.2.0'}
+
+sources = ['v%(version)s.tar.gz']
+source_urls = ['https://github.com/sib-swiss/pftools3/archive/refs/tags/']
+checksums = ['1de33d6444015942afa690d54a103ad5a3d68f416fad63f247c12fa3f71a550c']  # pftools3-3.2.12.tar.gz
+
+builddependencies = [
+    ('binutils', '2.37'),
+    ('CMake', '3.21.1'),
+    ('make', '4.3'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('libgd', '2.3.3'),
+    ('libharu', '2.3.0'),
+    ('libjpeg-turbo', '2.0.6'),
+    ('libpng', '1.6.37'),
+    ('PCRE2', '10.37'),
+    ('Perl', '5.34.0'),
+    ('zlib', '1.2.11'),
+]
+
+configure_cmd = 'cmake -DCMAKE_INSTALL_PREFIX:PATH=%(installdir)s -DCMAKE_BUILD_TYPE=Release '
+configure_cmd += '-DUSE_GRAPHICS=OFF -DUSE_PDF=ON '
+configure_cmd += '-DUSE_AFFINITY=OFF '  # Disable affinity for some old architectures
+configure_cmd += '../pftools3-%(version)s'
```

please use `configopts` for this rather than changing the `configure_cmd` directly
```suggestion
configopts = '-DUSE_GRAPHICS=OFF -DUSE_PDF=ON '
configopts += '-DUSE_AFFINITY=OFF '  # Disable affinity for some old architectures
```

---

Noted, done.